doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
Greg, I'd rather delete the section "I'm seeing warnings from GASNet about using a higher-performance network" from launchers.rst and never add it anywhere than put it in multilocale.rst.

I believe everything else in "Using the Portable UDP conduit" can be viewed as being about launchers.

I'd be happy to move "Using the Portabel UDP Conduit" to its own file and link to it from both launchers.rst and multilocale.rst.

I remain unconvinced that details (other than the basic outline of instructions) of the UDP conduit belong in multilocale.rst at all. I think multilocale.rst needs to be the outline and jumping off point for multilocale execution rather than something that gives one conduit much preferential treatment.